templates/nav.pug

```diff
@@ -36,6 +36,13 @@ mixin navdefaultlinks(page)
         li: a(href=loginDomain+"/login?dest=" + encodeURIComponent(baseUrl + page)) Login
         li: a(href=loginDomain+"/register") Register
 
+mixin navsuperadmin(page)
+  if superadmin
+    if page == '/'
```

Comment:
I think it'd make more sense for this to be a boolean `newTab` or something, since you're not actually doing anything with the page path.